src/weather_history_api/main.py

```
@@ -0,0 +1,88 @@
+
+    response = WeatherHistoryResponse(
+        location=LocationInfo(latitude=latitude, longitude=longitude),
+        range=DateRangeInfo(start_date=start_date, end_date=end_date),
+        days=days,
+        source="open-meteo",
+    )
+    return jsonify(_serialize_response(response))
+
+
+if __name__ == "__main__":
+    app.run(debug=True)
```

Ruff (0.15.9)

[error] Use of `debug=True` in Flask app detected

(S201)